sdk/python/runtime/extension.go

```diff
@@ -82,6 +82,32 @@ func (m *PythonSdk) UvVersion() string {
 	return m.Discovery.UserConfig().UvVersion
 }
 
+// Lets us determine if [tool.uv] index-url is set or not.
+func (m *PythonSdk) IsUvIndexUrlSpecified() bool {
+	return m.Discovery.UvConfig().IndexUrl != ""
+}
+
+// IndexUrl specified by [tool.uv] index-url from project's
+// pyproject.toml configuration if specified, otherwise
+// defaults to DefaultPackageIndexUrl.
+func (m *PythonSdk) IndexUrl() string {
+	if m.IsUvIndexUrlSpecified() {
+		return m.Discovery.UvConfig().IndexUrl
+	}
+	return DefaultPackageIndexUrl
+}
```

Comment:
This function is exposed to the API so it follows the API's convention for a description, instead of Go's.

```suggestion
// Uv's "index-url" setting
func (m *PythonSdk) IndexUrl() string {
	return m.Discovery.UvConfig().IndexUrl
}
```